### DISCUSSION TITLE
更新日志 - 22.03.6 - 20240628

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1674] [Announcements]
# 系统
1. mt7921u 驱动支持 Comfast CF-952AX（MT7921U） #1662
2. 升级时清理系统version文件，防止意外修改的version文件影响之后在线升级的版本判断 #1656

PS：国内拉取Docker镜像失败，可安装iStore中近期上架的“iStore增强”插件解决。如果是国外用户或者有自己的方式解决网络问题请勿安装，可能反而降低镜像拉取速度。（与此次固件更新无关）

# 应用
* QuickStart：Docker迁移向导取消自动设置国内USTC镜像仓库，因为USTC镜像仓库失效了。可根据前文解决Docker拉取镜像问题。
* iStore：升级到 v0.1.20-0 ，修复上一版本安装部分应用报500错误 linkease/istore#131

# 设备
* Radxa E20C： 修改USER按钮功能为重启/恢复出厂设置（短按重启，长按恢复出厂设置）
* Radxa E52C： 新增。此硬件基于RK3582。

**Comment by rayangl** [comment]
e52c啥时候上线

**Reply by jjm2473** [reply]
e52c代码已经发布了，固件没发，有需要可以自己编译。况且radxa不是还在调整硬件吗？硬件都还没发布吧。
